theme/src/css/_zendesk-variables.scss

```diff
@@ -40,15 +40,16 @@ $font-family-base: "Source Sans Pro", "Source Serif Pro", -apple-system,
 $font-weight-light: 200 !default;
 $font-weight-semi-bold: 600 !default;
 
-$normal-font-size: 1rem !default; // 16px
-$small-font-size: 0.75rem !default; // 12px
+$font-size-default: 1rem !default; // 16px
+$font-size-md: 0.875rem !default; // 14px
```

Comment:
Hey @antoniorhee Why are we introduce `$font-size-md` to Zendesk if it's never used?